### PR TITLE
[SMALLFIX] Fix Findbugs issues with the Mesos framework

### PR DESCRIPTION
**Pull request by aaudiber**

### CODE REVIEW

**Comment by haoyuan** [comment]
@apc999 @calvinjia @jsimsa

File: integration/mesos/src/main/java/tachyon/mesos/TachyonFramework.java [hunk 1]
```diff
@@ -95,10 +96,11 @@ public void reregistered(SchedulerDriver driver, Protos.MasterInfo masterInfo) {
 
     @Override
     public void resourceOffers(SchedulerDriver driver, List<Protos.Offer> offers) {
-      double masterCpu = sConf.getInt(Constants.INTEGRATION_MASTER_RESOURCE_CPU);
-      double masterMem = sConf.getBytes(Constants.INTEGRATION_MASTER_RESOURCE_MEM) / Constants.MB;
-      double workerCpu = sConf.getInt(Constants.INTEGRATION_WORKER_RESOURCE_CPU);
-      double workerMem = sConf.getBytes(Constants.INTEGRATION_WORKER_RESOURCE_MEM) / Constants.MB;
+      // TODO(andrew): Should these be doubles or longs?
```
Comment (apc999) [inline integration/mesos/src/main/java/tachyon/mesos/TachyonFramework.java]: it is the number of CPU cores, I think `int` should be fine

**Comment by apc999** [comment]
+1 LGTM. Thanks for fixing developer build!

File: integration/mesos/src/main/java/tachyon/mesos/TachyonFramework.java (hunk 1 above)
Comment (jsimsa) [inline integration/mesos/src/main/java/tachyon/mesos/TachyonFramework.java]: Int/long should be fine. There used to be old logic that required support for fractional values. But that's gone now.

File: integration/mesos/src/main/java/tachyon/mesos/TachyonFramework.java [hunk 2]
```diff
@@ -245,7 +247,7 @@ public void slaveLost(SchedulerDriver driver, Protos.SlaveID slaveId) {
     public void statusUpdate(SchedulerDriver driver, Protos.TaskStatus status) {
       String taskId = status.getTaskId().getValue();
       Protos.TaskState state = status.getState();
-      System.out.printf("Task %s is in state %s\n", taskId, state);
+      System.out.printf("Task %s is in state %s%n", taskId, state);
```
Comment (jsimsa) [inline integration/mesos/src/main/java/tachyon/mesos/TachyonFramework.java]: This does not seem right.

Comment (aaudiber) [inline integration/mesos/src/main/java/tachyon/mesos/TachyonFramework.java]: `%n` produces the platform-specific newline character, while `\n` is unix-specific.

**Pull request merged**